sdk/state/state.go

```diff
@@ -18,18 +18,26 @@ const (
 )
 
 type Channel struct {
-	Status ChannelStatus
-
+	Status                 ChannelStatus
 	Initiator              bool
 	InitiatorEscrowAccount *keypair.FromAddress
 	ResponderEscrowAccount *keypair.FromAddress
 
+	// TODO - is this the best way to rep Balance? (perspective of initiator/responder and not Me/Other)
```

Comment:
I tried using subjective for Balance but it was annoying since payments are objective. I'll take another look, maybe something that just has to be dealt with, or possibly payments could be subjective 🤔